src/Source/DoctrineORMQuerySourceIterator.php

```diff
@@ -155,7 +155,7 @@ protected function getValue($value)
     {
         if (is_array($value) || $value instanceof \Traversable) {
             $value = null;
-        } elseif ($value instanceof \DateTime) {
+        } elseif ($value instanceof \DateTime || $value instanceof \DateTimeInterface) {
```

Comment:
It is useful to write it like this because \DateTimeInterface does not exist for old versions of php. To be dropped later though.

Reply:
Yes, I added this condition to compatibility with old versions of php

Reply:
Good thinking, I just wanted to clarify that for other reviewers.

Reply:
Does it make sense now that we have dropped php up to 7.1?

Reply:
Nope, please remove it